Makefile

```diff
@@ -3,10 +3,9 @@ all:
 
 .PHONY: test clean
 test:
-	./node_modules/.bin/mocha
+	npm run test
 
 circleci:
-	NODE_ENV=test ./node_modules/.bin/mocha --require blanket --reporter mocha-lcov-reporter | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
Can't we keep coverall without mocha-lcov-reporter if the issue is only from the reporter?

Reply:
I'll try to find workaround.